monai/application/handlers/stats_handler.py

```diff
@@ -0,0 +1,110 @@
+import traceback
+from .handler import Handler
+
+
+class StatsHandler(Handler):
+    """StatsHandler defines a set of Ignite Event-Handlers for all the log printing logics.
+    It's can be used for any Ignite Engine(trainer, validator and evaluator).
+    And it can support logging for epoch level and iteration level with pre-defined StatsLoggers.
+
+    Args:
+        epoch_print_logger (Callable): customized callable printer for epoch level logging.
+                                          must accept parameter "engine", use default printer if None.
+        iteration_print_logger (Callable): custimized callable printer for iteration level logging.
+                                         must accept parameter "engine", use default printer if None.
+
+    """
+    def __init__(self,
+                 epoch_print_logger=None,
+                 iteration_print_logger=None):
+        Handler.__init__(self)
+        self.epoch_print_logger = epoch_print_logger
```

Comment:
using `logging.logger` object (with streamhandlers/file handlers) from python to do printing instead of `print`